recipes/arrow/all/conanfile.py

```diff
@@ -101,7 +101,7 @@ class ArrowConan(ConanFile):
         "with_glog": False,
         "with_grpc": False,
         "with_json": False,
-        "with_thrift": False,
+        "with_thrift": True,
```

Comment:
Is there a good reason to have this set to true? Is this _just_ for testing the changes n the CI? We could then revert it now

Reply:
Yes! `with_thrift` needs to be True in order to build Arrow with Parquet support (which I have enabled based on https://github.com/conan-io/conan-center-index/pull/24044#issuecomment-2121953837).